index.html

```diff
@@ -229,28 +265,25 @@ <h4>Skills</h4><img class="arrow" src="images/right.png" alt="">
     <!-- contact form -->
     <section class="contact_form">
         <div class="contact_wrapper">
-            <h1>Contact me</h1>
-            <p>
-                If you have an application you are interested in developing,
-                a feature that you need built or a project that needs coding.
-                I’d love to help with it
-            </p>
-            <form action="https://formspree.io/f/xbjeloeq" method="post">
-                <ul>
-                    <li>
-                      <input name="username" type="text" placeholder="Your name" maxlength="30"required>
-                    </li>
-                    <li>
-                      <input name="email" type="email" placeholder="youremail@example.com" required>
-                    </li>
-                    <li>
-                      <textarea name="message" id="message" placeholder="Write your message here" maxlength="500" required></textarea>
-                    </li>
-                </ul>
-                <input type="submit" value="Get It Touch">
-            </form>
+            <div class="form_wrapper">
+                <h1>Contact me</h1>
+                <p>
+                    If you have an application you are interested in developing,
+                    a feature that you need built or a project that needs coding.
+                    I’d love to help with it
+                </p>
```

Comment:
- [ ] Great job making all the changes suggested by the previous reviewer :muscle: You only missed this one. 
- [ ] Just like the previous CR mentioned before, I would also suggest that you add a `width: 569px` to this (`p` tag) as shown in the Figma design guidelines below to get the desired look we're aiming for :wink: 
- Figma Design Guide
![image](https://user-images.githubusercontent.com/102699958/213846717-d6401c3a-5249-4db9-ae92-c19f10fae0f9.png)

- [ ] I also noticed that there was an added margin to your `contact me` section and now it looks quite different from the Figma design. I believe this was introduced in your new commit as it didn't look like this the first time you sent it for review so I would suggest you revert back to the old styles for this part or redo it to fit the Figma design :smile: 
- Your Design now
- [ ] You can notice that there is the extra spacing between the blue border and the `Contact me` header.
![image](https://user-images.githubusercontent.com/102699958/213846617-5c1d5d91-de1b-48e5-8c97-c0700beeb890.png)
- The  Figma Design
![image](https://user-images.githubusercontent.com/102699958/213847201-3107effb-a3eb-40e2-bdef-f469518157bf.png)